src/runtime_src/core/common/api/aie/xrt_graph.cpp

```diff
@@ -228,6 +228,76 @@ send_exception_message(const char* msg)
   xrt_core::message::send(xrt_core::message::severity_level::error, "XRT", msg);
```

Comment:
Copyright needs to be updated (line 3).  Being that we are part of AMD now, the following addition should be made.
`Copyright (C) Advanced Micro Devices, Inc.`